Weights/examples/Weights/projection_traits.cpp

```diff
@@ -0,0 +1,49 @@
+#include <CGAL/Simple_cartesian.h>
+#include <CGAL/Weights/utils.h>
+#include <CGAL/Weights/wachspress_weights.h>
+#include <CGAL/Weights/mean_value_weights.h>
+#include <CGAL/Projection_traits_xy_3.h>
+
+// Typedefs.
+using Kernel   = CGAL::Simple_cartesian<double>;
+using FT       = typename Kernel::FT;
+using Point_3  = typename Kernel::Point_3;
+using Vector_3 = typename Kernel::Vector_3;
+using PTraits  = CGAL::Projection_traits_xy_3<Kernel>;
+
+int main() {
+
+  // Create a polygon and a query point.
+  const std::vector<Point_3> polygon =
+    { Point_3(0, 0, 1), Point_3(1, 0, 1), Point_3(1, 1, 1), Point_3(0, 1, 1) };
+
+  const FT half = FT(1) / FT(2);
```

Comment:
This is an example.  In the example the FT is double so no need to write `FT(1)/FT(2)`.    We do that inside CGAL to be generic, but here it makes no sense.

Reply:
Ok, fixed.